lib/validation.tsx

```diff
@@ -106,8 +107,16 @@ const isFielddResponseValid = (
       break;
     }
     case "fileInput": {
-      const typedValue = value as Record<string, string>;
-      if (validator.required && typedValue["file"] === null) return t("input-validation.required");
+      const typedValue = value as { file: File; src: FileReader; name: string; size: number };
```

Comment:
I just quickly refactored to type `value` with what is output from the fileInput component instead of using `Record<string, unkown>`